core/src/main/java/org/apache/iceberg/PartitionsTable.java

```diff
@@ -49,6 +49,16 @@ public class PartitionsTable extends BaseMetadataTable {
         new Schema(
             Types.NestedField.required(1, "partition", Partitioning.partitionType(table)),
             Types.NestedField.required(4, "spec_id", Types.IntegerType.get()),
+            Types.NestedField.required(
+                9,
+                "last_updated",
```

Comment:
nit: how about 'last_updated_ms'?   (matches Table metadata and also as below, we have a suffix for snapshot_id)